meta-mender-core/recipes-mender/mender/mender.inc

```diff
@@ -10,7 +10,7 @@ def cert_location_if_server_crt_in(src_uri, d):
     return ""
```

Comment:
I was looking back at forth at this function. But I agree that we shall not modify this, so that a production cert is still installed in `/etc/mender/server.crt` :+1: